package.json

```diff
@@ -48,13 +49,15 @@
 		"file"
 	],
 	"dependencies": {
-		"is-docker": "^2.0.0",
-		"is-wsl": "^2.1.1"
+		"aggregate-error": "^3.1.0",
+		"define-lazy-prop": "^2.0.0",
```

Comment:
More dependencies 😢

Reply:
To remove `define-lazy-prop` we'll either need to come up with a new way to provide the data or not cache it. Otherwise, we would just be reinventing the wheel.

To remove `aggregate-error` we'll need to throw a different type of error.

Reply:
I don't see it as a problem. `aggregate-error` will eventually go away when we can use the native one.

The other one is tiny: https://bundlephobia.com/result?p=define-lazy-prop@2.0.0